distribution/docs/pom.xml

```diff
@@ -381,6 +381,23 @@
                     <skip>true</skip>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
❓ Is there a reason you used the failsafe plugin instead of the surefire plugin? I think this will cause the tests to run in the integration test phase instead of the normal test phase that unit tests typically run in.

Reply:
I think I misunderstood what was happening since it seemed to run at the appropriate time in the logs while testing this change. I can update to surefire and verify again.